src/wallet/wallet.cpp

```diff
@@ -4608,36 +4606,48 @@ CWalletKey::CWalletKey(int64_t nExpires)
     nTimeExpires = nExpires;
 }
 
-void CMerkleTx::SetMerkleBranch(const uint256& block_hash, int posInBlock)
+void CMerkleTx::SetMerkleBranch(const uint256& block_hash, int posInBlock, int block_height)
 {
     // Update the tx's hashBlock
     hashBlock = block_hash;
 
     // set the position of the transaction in the block
     nIndex = posInBlock;
+
+    // set tx height
+    m_block_height = block_height;
 }
 
-int CMerkleTx::GetDepthInMainChain(interfaces::Chain::Lock& locked_chain) const
+int CMerkleTx::GetDepthInMainChain() const
 {
+    if (pwallet == nullptr)
+        return 0;
+    assert(m_block_height >= -1);
```

Comment:
This line is introduced in a too-early commit (_Add m_last_block_processed_height field in CWallet_) where the member variable _m_block_height_ doesn't yet exist. The _Add m_last_block_processed_height field in CWallet_ commit doesn't build because the variable is not declared.

Reply:
Ooops, bad rebase sorry, should be corrected